packages/commons/src/auth/converters.ts

```diff
@@ -1,13 +1,7 @@
-import {
-  KeyReadModel,
-  JWKKeyInReadModel,
-  missingRequiredJWKClaim,
-} from "pagopa-interop-models";
+import { JWKKey, missingRequiredJWKClaim, Key } from "pagopa-interop-models";
 import { createJWK, decodeBase64ToPem } from "./jwk.js";
 
-export const fromKeyToReadModelJWKKey = (
-  key: KeyReadModel
-): JWKKeyInReadModel => {
+export const keyToJWKKey = (key: Key): JWKKey => {
```

Comment:
Changes here:
- rename the function
- change the input type, because it's not necessary to involve `KeyReadModel` (which is the way keys are stored in a client, whereas this converter is for the `keys` collection itself)
- the returned type is the same, but it has been renamed.